raiden/network/proxies/token_networks_registry.py

```diff
@@ -0,0 +1,199 @@
+    def add_token(self, token_address):
+        if not isaddress(token_address):
+            raise ValueError('token_address must be a valid address')
+
+        log.info(
+            'add_token called',
+            node=pex(self.node_address),
+            token_address=pex(token_address),
+            registry_address=pex(self.address),
+        )
+
+        transaction_hash = estimate_and_transact(
+            self.proxy,
+            'createERC20TokenNetwork',
+            token_address
+        )
+
+        self.client.poll(unhexlify(transaction_hash), timeout=self.poll_timeout)
+        receipt_or_none = check_transaction_threw(self.client, transaction_hash)
+        if receipt_or_none:
+            log.info(
+                'add_token failed',
+                node=pex(self.node_address),
+                token_address=pex(token_address),
+                registry_address=pex(self.address),
+            )
+            raise TransactionThrew('createERC20TokenNetwork', receipt_or_none)
+
+        manager_address = self.manager_address_by_token(token_address)
+
+        if manager_address is None:
+            log.info(
+                'add_token failed and check_transaction_threw didnt detect it',
+                node=pex(self.node_address),
+                token_address=pex(token_address),
+                registry_address=pex(self.address),
+            )
+
+            raise RuntimeError('token_to_token_networks failed')
+
+        log.info(
+            'add_token sucessful',
+            node=pex(self.node_address),
+            token_address=pex(token_address),
+            registry_address=pex(self.address),
+            manager_address=pex(manager_address),
+        )
+
+        return manager_address
+
+    def tokenadded_filter(self, from_block=None, to_block=None):
+        topics = [CONTRACT_MANAGER.get_event_id(EVENT_TOKEN_ADDED2)]
+
+        registry_address_bin = self.proxy.contract_address
+        filter_id_raw = new_filter(
+            self.client,
+            registry_address_bin,
+            topics,
+            from_block=from_block,
+            to_block=to_block,
+        )
+
+        return Filter(
+            self.client,
+            filter_id_raw,
+        )
+
+    def manager(self, manager_address):
+        """ Return a proxy to interact with a TokenNetwork. """
+        if not isaddress(manager_address):
+            raise ValueError('manager_address must be a valid address')
+
+        if manager_address not in self.address_to_channelmanager:
+            manager = TokenNetwork(
+                self.client,
+                manager_address,
+                self.poll_timeout,
+            )
+
+            token_address = manager.token_address()
+
+            self.token_to_channelmanager[token_address] = manager
+            self.address_to_channelmanager[manager_address] = manager
+
+        return self.address_to_channelmanager[manager_address]
+
+    def manager_by_token(self, token_address):
```

Comment:
Same as above ^ : 

Since we no longer use channel managers rename this function, its dosctrings, its error string and its vars to correspond to token network.